src/resources/subscription_ext.rs

```diff
@@ -39,3 +40,105 @@ impl CreateSubscriptionItems {
         }
     }
 }
+
+/// The resource representing a Stripe "SubscriptionSchedule".
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct SubscriptionSchedule {
```

Comment:
Is this not auto-generated?   Is it maybe in a new file that can be added to `openapi/src/commit`?

Unsure why it is in `SubscriptionExt`.